components/director/internal/tenantmapping/staticuser.go

```diff
@@ -0,0 +1,55 @@
+package tenantmapping
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+
+	"github.com/pkg/errors"
+)
+
+type StaticUser struct {
+	Username string   `json:"username"`
+	Tenant   string   `json:"tenant"`
```

Comment:
can we use UUID instead of string for tenant? I am wondering what will happen if someone provide invalid tenant, maybe we should fail fast on loading data to the memory?